Goal: slapd.conf template fixes

- DB_CONFIG ownership
- load modules
- path of kolab2.schema
- enable horde schema

Author: Peter Eisentraut <petere@debian.org>

Upstream status: Specific to Debian, parts can be included

Index: b/templates/DB_CONFIG.slapd.template.in
===================================================================
--- a/templates/DB_CONFIG.slapd.template.in
+++ b/templates/DB_CONFIG.slapd.template.in
@@ -1,7 +1,7 @@
 KOLAB_META_START
 TARGET=@ldapserver_dir@/DB_CONFIG
 PERMISSIONS=0640
-OWNERSHIP=@kolab_musr@:@kolab_rgrp@
+OWNERSHIP=@ldapserver_usr@:@ldapserver_grp@
 KOLAB_META_END
 # (c) 2005 Klaraelvdalens Datakonsult AB
 #     Written by Steffen Hansen <steffen@klaralvdalens-datakonsult.se>
Index: b/templates/slapd.conf.template.in
===================================================================
--- a/templates/slapd.conf.template.in
+++ b/templates/slapd.conf.template.in
@@ -13,6 +13,12 @@
 # this file is automatically written by the Kolab config backend and should have the 
 # file mode 0640
 
+modulepath	/usr/lib/ldap
+moduleload	back_bdb
+moduleload	back_monitor
+moduleload	refint
+moduleload	unique
+
 # manual additions are lost unless made to the template in the Kolab config directory
 # the template is  @sysconfdir@/kolab/templates/slapd.conf.template
 
@@ -20,8 +26,8 @@
 include @ldapserver_schemadir@/cosine.schema
 include @ldapserver_schemadir@/inetorgperson.schema
 include @ldapserver_schemadir@/rfc2739.schema
-include @ldapserver_schemadir@/kolab2.schema
-#include @ldapserver_schemadir@/horde.schema
+include @ldapserver_schemadir@/kolab2.schema
+include @ldapserver_schemadir@/horde.schema
 
 pidfile		@ldapserver_pidfile@
 argsfile	@ldapserver_argsfile@
